packages/fuels-test-helpers-bin/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "fuels-test-helpers-bin"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/fuels-rs"
+description = "Fuel Rust SDK test helpers using fuel-core.bin."
+
```

Is this meant to be a published crate?
```suggestion
publish = false
```